distribution/docs/src/main/jdocs/content/_plugins/event-processor-contents.adoc

```diff
@@ -0,0 +1,77 @@
+:type: plugin
+:status: published
+:title: Event Processor
+:link: _event_processor
+:plugintypes: postingest
+:summary: Creates, updates, and deletes subscriptions.
+
+The Event Processor provides an engine that creates, updates, and deletes subscriptions for event notification.
+These subscriptions optionally specify a filter criteria so that only events of interest to the subscriber are posted for notification.
+
+An internal subscription tracker monitors the OSGi registry, looking for subscriptions to be added (or deleted).
+When it detects a subscription being added, it informs the Event Processor, which sets up the subscription's filtering and is responsible for posting event notifications to the subscriber when events satisfying their criteria are met.
+
+The Standard Event Processor is an implementation of the Event Processor and provides the ability to create/delete subscriptions.
+Events are generated by the ${ddf-catalog}Framework as metacards are created/updated/deleted and the Standard Event Processor is called since it is also a Post-Ingest Plugin.
+The Standard Event Processor checks each event against each subscription's criteria.
+
+When an event matches a subscription's criteria the Standard Event Processor:
+
+* invokes each pre-delivery plugin on the metacard in the event.
+* invokes the Delivery Method's operation corresponding to the type of event being processed, e.g., created operation for the creation of a metacard.
+
+===== Installing the Event Processor
+
+The Event Processor is installed by default with a standard installation in the ${ddf-catalog} application.
+
+===== Configuring the Event Processor
+
+The Event Processor has no configurable properties.
+
+===== Event Processing and Notification
+
+As metacards are created, updated, and deleted, the Catalog's Event Processor is invoked (as a post-ingest plugin) for each of these events.
+TheEvent Processor applies the filter criteria for each registered subscription to each of these ingest events to determine if they match the criteria.
+If an event matches a subscription's criteria, any pre-delivery plugins that are installed are invoked, the subscription's Delivery Method is retrieved, and its operation corresponding to the type of ingest event is invoked. 
+For example, the DeliveryMethod's `created()` function is called when a metacard is created.
+The Delivery Method's operations subsequently invoke the corresponding operation in the client's event consumer service, which is specified by the callback URL provided when the Delivery Method was created.
```

Comment:
✏️  If we have any documentation on how subscriptions are created (e.g., using CSW), we may want to refer to that section.